### DISCUSSION TITLE
I can even run League of Legends on 3500u hackbook!

### DISCUSSION THREAD

**Discussion by ChitoKim** [ChefKissInc/NootedRed#177] [Show and tell]
![Screen Shot 2023-09-23 at 3 17 14 AM](https://github.com/ChefKissInc/NootedRed/assets/76689397/92fedfc2-a45c-4bde-90a8-4b371297250f)
Every graphics work flawlessly. Though I get  frame drops due to the lack of cooling of my laptop, I could finish the game. I'm content because league sucks even on a real Mac. This thing hadn't been even conceivable.. Thank you, ChefKissInc!

**Comment by revogabe** [comment]
My processor is a Ryzen 5 5600g, although my Hackintosh is reasonably stable, I had difficulties getting League of Legends to run, when opening the client the screen freezes and the mouse becomes unusable.

How did you manage to get this to run?

**Reply by ChitoKim** [reply]
Yeah, I also have crackling graphics and freezes on its client. The game itself runs okay. Well, League client is a piece of shit even on Windows. 
I removed SMCAMDProcessor.kext , AMDRyzenPowerManagement.kext and replaced them by SMCProcessorAMD.kext. Maybe that helped. I still get random cracks and freezes, but better now.

**Reply by ChitoKim** [reply]
Plus, I think that the crash is the issue of OpenGL. You know, OpenGL behaves terribly with NootedRed now. As I changed the in-game renderer to Metal, the performance greatly improved. However, maybe the client is running on OpenGL. I suspect that because when I ran league on linux via wine, the client and in-game had distinct rendering API settings.  So, client still crashes, freezes, and flashes like shit.
If you're trying to switch to Metal, try the link below
https://www.reddit.com/r/macgaming/comments/tiqv96/league_of_legends_metal_double_fps/
